### DISCUSSION TITLE
Unable to Log In with Provided Credentials

### DISCUSSION THREAD

**Discussion by Bhargav983** [egovernments/Digit-Core#546] [Sandbox]
Dear Support Team,

I am trying to log in to the DIGIT demo platform for the first time, but I am unable to do so using the credentials provided. Below are the details of my issue:

URL: https://staging.digit.org/employee
Username/Employee ID: [Your Employee ID]

password is provided but no idea on what to enter at username or empId 

gave multiple combinations as inouts - no use 

Please assist me in resolving this issue as soon as possible so I can access the platform.

Thank you for your prompt support.

**Comment by pradeepkumarcm-egov** [comment]
Hi Bhargav,

 Register your email id [here](https://egov.org.in/access-digit/).

 You will receive email   "User Credentials for DIGIT Instance" with user information.
 
The user credentials for different DIGIT applications are shared in table format. The usernames are created and marked in first column of the table.

Please select 'City B' in Login Screen  

Login url for DIGIT Instance is - https://staging.digit.org/employee

**Reply by Bhargav983** [reply]
We have recieved the mail  with password but no employee id or username which is required for login.

**Reply by Bhargav983** [reply]
Request you to  look into this and gudie us further

**Reply by vinothrallapalli-eGov** [reply]
@Bhargav983 The table first row of the received email contains the names of the modules, the user name for login, and the corresponding roles mapped for each module in the corresponding column.

**Reply by Bhargav983** [reply]
that is not working , is it possible to get onto a google meet or zoom to resolve this issue , 

it will be really a great help

**Reply by vinothrallapalli-eGov** [reply]
@Bhargav983 We have confirmed that the new registration process works correctly with the provided credentials. The email message has been improved with more detailed information. Kindly register again and give it another try.
